lib/testing/src/lib/process-services-cloud/pages/task-form-cloud-component.page.ts

```diff
@@ -42,6 +42,10 @@ export class TaskFormCloudComponent {
         await BrowserVisibility.waitUntilElementIsNotVisible(this.completeButton);
     }
 
+    async completeButtonIsEnabled() {
```

Comment:
missing return value type

Reply:
Suggest this name: isCompleteButtonEnabled